notebooks/20220114_look_at_sra_org_composition.ipynb

```diff
@@ -0,0 +1,874 @@
+{
```

Comment:
<p>Are the incorrect labels caused by typos? Or is it an input error in the dataset?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-microberna/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>That's a great question. It could be caused by typos, but what's more likely is that the submitters are using a non-standard, or NCBI-specific, taxonomy identifier, and I'm matching it against the GTDB taxonomy. This analysis gives a good approximation of cover/missingness between GTDB and NCBI, but subsequent things I'll do after running the pipeline will give much more accurate estimates. </p>